### PR TITLE
feat(template): generator template for packages

### PR DESCRIPTION
**Pull request by barsukov**
# Summary
Add a nice generator that use turbo prompting system to generate packages inside of our repo.

# Changes Made

- Template project for packages support all latest frameworks
- rpm run generate:package task in root of directory
-  Readme update
# Related Issues

- #199 

# Screenshots (if applicable)

# Testing Instructions

1. `npm i`
2. `npm run generate:package NAME`

# Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [ ] My changes generate no new warnings or errors.

### CODE REVIEW

File: packages/template/README.md [hunk 1]
```diff
@@ -2,84 +2,36 @@
...
-```bash
-npm -w template run test
-```
-
-Alternatively, from within the workspace:
+3. Run the following command to install the **Template** package as a global dependency:
 
 ```bash
-wb npm -w template run test
+turbo gen workspace --type package --copy @cloudoperators/juno-package-template
```
Comment (andypf) [inline packages/template/README.md:24]: don't you want to reference here to the task `generate:package`?

Comment (barsukov) [reply packages/template/README.md:24]: This is on package lvl is more correct, that task that you suggested is more on root lvl

**Review by andypf — APPROVED** [review]

**Review by TilmanHaupt — APPROVED** [review]

File: docs/getting-started.md [hunk 2]
```diff
@@ -60,6 +60,12 @@ Before you begin, ensure you have the following software installed on your machi
      npm run pkg @cloudoperators/juno-app-example
      ```
 
+   - **Generator:Package**
+     The npm run generate:package command is used to generate a new package in a project. When running this command, you need to provide a name for the package that starts with `@cloudoperators/juno-`.
```
Comment (ArtieReus) [inline docs/getting-started.md:64]: Would be a good idea to describe here what is a package and what would be the use case?

File: packages/template/README.md [hunk 3]
```diff
@@ -2,84 +2,36 @@
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
 
-**Template** is a [brief description of your library's purpose].
+**Template** is a package that allows you to generate new packages with ease. It provides a convenient command-line interface for generating packages based on predefined templates.
```
[the remark this answers is not in the document]
Comment (barsukov) [reply packages/template/README.md:5]: what do you mean?

**Pull request merged**